utils/post_utils.jsx

```diff
@@ -101,7 +101,9 @@ export function canEditPost(post, editDisableAction) {
     const isOwner = isPostOwner(post);
     canEdit = haveIChannelPermission(store.getState(), {channel: post.channel_id, team: channel && channel.team_id, permission: Permissions.EDIT_POST});
     if (!isOwner) {
-        canEdit = canEdit && haveIChannelPermission(store.getState(), {channel: post.channel_id, team: channel && channel.team_id, permission: Permissions.EDIT_OTHERS_POSTS});
+        // I only can edit my posts, at least until phase-2
```

Comment:
Was the server actually allowing editing of posts? Is the only thing gating this a UI change? Do we need to make server-side changes until phase-2 is finished?

Reply:
Nope, actually is on purpose to do it only in the interface. To maintain the current 4.8 API behavior after the migration to 4.9.